composer-local.json

```diff
@@ -3,9 +3,13 @@
   "description": "Greenpeace Planet 4 Ukraine",
   "license": "MIT",
   "require": {
-    "greenpeace/planet4-child-theme-ukraine": "dev-main"
+    "greenpeace/planet4-child-theme-ukraine": "dev-main",
+    "plugins/sitepress-multilingual-cms": "*",
+    "plugins/wpml-string-translation": "*",
+    "plugins/wpml-elasticpress": "*"
   },
   "scripts": {
-    "site:custom" : []
+    "core:language": "wp language core install uk_UA",
```

Comment:
You can also activate the language pack in the same command. See similar example [here](https://github.com/greenpeace/planet4-greece/blob/main/composer-local.json#L13).

I haven't tested it, but is it `uk_UA` or just `uk`, as in the [WP locale list](https://make.wordpress.org/polyglots/teams/)?

Reply:
I've tried adding it to the Canadian dev site languages, and the string was `uk_UA` 🤔

Reply:
If I run:

```bash
npx wp-env run cli wp language core install uk_UA
```

I get: `Warning: Language 'uk_UA' not available.`


Maybe the official locale is `uk_UA` but the WP language package is just `uk`.